packages/devtools_app/integration_test/test/live_connection/debugger_panel_test.dart

```diff
@@ -147,6 +148,26 @@ void main() {
     // Verify that line 40 is focused:
     expect(isLineFocused(line40Finder), isTrue);
   });
+
+  test('invalid setBreakpoint throws exception', () async {
```

Comment:
This test case will fail when connected to a web app (so once https://github.com/flutter/devtools/pull/5917 is in), DWDS doesn't throw the same error. I'm curious why we are testing the vm service implementation for `addBreakpoint`? Another option might be to test that if we try to add an invalid breakpoint, the breakpoint isn't added in DevTools.

Reply:
This was an existing test in service_manager_test.dart that I moved over. If this is already test covered in the vm service, happy to remove.

Reply:
Here is an equivalent test in the `vm_service`: https://github.com/dart-lang/sdk/blob/b5c42b8cc6d26a0f840fae5b7cbae298dbf7d182/pkg/vm_service/test/add_breakpoint_rpc_kernel_test.dart#L153

And in DWDS: https://github.com/dart-lang/webdev/blob/1551fe6c9faa010b89de241e253d697f1b405f02/dwds/test/chrome_proxy_service_test.dart#L106

So I think this should be safe to remove!